src/components/Counter.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import './Counter.scss';
+
+type State = {
+  counter: number;
+};
+
+export class Counter extends React.Component<{}, State> {
+  state = {
+    counter: 0,
+  };
+
+  addOne = () => {
+    this.setState((prevState) => ({
+      counter: prevState.counter + 1,
+    }));
+  };
+
+  add100 = () => {
+    this.setState((prevState) => ({
+      counter: prevState.counter + 100,
+    }));
+  };
+
+  increase = (counter: number) => {
+    if (counter % 5 === 0) {
+      this.addOne();
+      this.add100();
+    } else {
+      this.addOne();
+    }
```

Comment:
```suggestion
    if (counter % 5 === 0) {
      this.add100();
    } 
    
    this.addOne();
```
Ты в любом случае один плюсуешь